### DISCUSSION TITLE
Next Config for version 13.4

### DISCUSSION THREAD

**Discussion by eric-sison** [vercel/next.js#49272] [App Router]
The next.config.js still has `experimental: { appDir: true }`
I tried to remove it, and it still works. 
Is there any particular reason why it's still there, despite going out of beta?

**Comment by icyJoseph** [comment]
Well, for some reason, there's a bug now in the pages dir, and using this fixes it 😅 

```js
experimental: {
  appDir: false,
},
```

But yeah it'll probably be removed soon enough. Or maybe moved to the root for those who want to totally opt-out.

**Reply by eric-sison** [reply]
Could you tell me about the current bug in the pages dir, or point me to the bug report so I can read more about it? I have only just started using the latest version, and haven't explored much yet. :)

**Reply by vinnymac** [reply]
`13.4.3` fixed the pages dir issues for me, so `appDir: true` no longer breaks my application.

This was the [specific change](https://github.com/vercel/next.js/pull/49805) that fixed it for me.

**Comment by eric-sison** [comment]
By the way, I just saw a PR (#49241), trying to resolve this issue.

**Reply by icyJoseph** [reply]
yeah there's quite a lot of discussions and bug reports I think, https://github.com/vercel/next.js/issues/49261

**Reply by icyJoseph** [reply]
They merged the removal of the experimental app dir flag to canary.

As for the deploy to Vercel bug: https://github.com/vercel/next.js/pull/49274
